server.js

```diff
@@ -1,10 +1,13 @@
-const express = require("express");
+const fallback = require('express-history-api-fallback');
+const express = require('express');
 const app = express();
+const path = require('path');
+const root = path.resolve(__dirname, 'dist');
+app.use(express.static(root));
+app.use(fallback('index.html', { root: root }));
+
 const PORT = 3000;
 process.env.PORT || PORT;
```

Comment:
`process.env.PORT || PORT` Сейчас никуда не присваивается
```suggestion
const PORT = process.env.PORT || 3000;
```